core-web/apps/dotcms-block-editor/src/app/dot-block-editor/dot-block-editor.component.ts

```diff
@@ -42,42 +45,91 @@ export class DotBlockEditorComponent implements OnInit {
     constructor(private injector: Injector, public viewContainerRef: ViewContainerRef) {}
 
     ngOnInit() {
-        this.editor = new Editor({
-            extensions: [
-                StarterKit,
-                ContentletBlock(this.injector),
-                ImageBlock(this.injector),
-                ActionsMenu(this.viewContainerRef),
-                DragHandler(this.viewContainerRef),
-                ImageUpload(this.injector, this.viewContainerRef),
-                BubbleLinkFormExtension(this.injector, this.viewContainerRef),
-                DotBubbleMenuExtension(this.viewContainerRef),
-                // Marks Extensions
-                Underline,
-                TextAlign.configure({ types: ['heading', 'paragraph', 'listItem', 'dotImage'] }),
-                Highlight.configure({ HTMLAttributes: { style: 'background: #accef7;' } }),
-                Link.configure({ openOnClick: true }),
-                Placeholder.configure({
-                    placeholder: ({ node }) => {
-                        if (node.type.name === 'heading') {
-                            return `${toTitleCase(node.type.name)} ${node.attrs.level}`;
-                        }
-
-                        return 'Type "/" for commmands';
+        const defaultExtensions: Extensions = [
+            DotConfigExtension({
+                lang: this.lang,
+                allowedContentTypes: this.allowedContentTypes,
+                allowedBlocks: this.setAllowedBlocks()
```

Comment:
You can do what happens in the `setAllowedBlocks` if you create a setter in the `Input()`